public/index.pug

```diff
@@ -30,6 +30,12 @@ html(lang='en-US')
     )
     link(rel='license', href='license.txt')
     link(rel='canonical', href=pageUrl)
+    svg(xmlns='http://www.w3.org/2000/svg')
```

Comment:
Sorry, may have missed something in your comment. Should this have the `width="0"` and `height="0"` attributes?

Reply:
The SVG is hidden with a combination of `width="0"` and `height="0"` or just using `style="display:none"`, so it is the same. The SVG spec doesn't recommends any choice.

Reply:
Ups, seems that I've forgotten the `style="display:none"`. Fixed by #177